entities/auctioneer.py

```diff
@@ -28,6 +36,11 @@ def register(self, entity: Entity) -> int:
         registration_key = self._registration(entity.type)
         self.entities[entity.type][registration_key] = entity
 
+        if self.env.config.debug is True and self.region.id < 1:
```

Comment:
Why do you limit the region id?

_Never_ test against a normal object using `is`. It generally does not hold true, and should only be done to test against `None`. Here, `if self.env.config.debug and self.region.id < 1: ` is what you want.